docs/book/releases/v2.3.0.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.3.0 release
+
+## New Feature
+
+- CSI Migration for in-tree vSphere volumes is promoted to GA. This feature requires upgrading Kubernetes to 1.19 release.
+
+- XFS Filesystem support as an Alpha feature. Volumes can be provisioned by specifying `xfs` as the `fstype` in the [Storage Class](https://vsphere-csi-driver.sigs.k8s.io/features/block_volume.html).
+
+- Raw Block volume support as an Alpha feature. Visit the [Raw Block Volumes documentation](https://vsphere-csi-driver.sigs.k8s.io/features/raw_block_volume.html) for more details.
+
+## Notable Changes
+
+- Ability to set the maximum volumes that can be attached to a Node VM. This can be modified in the [CSI 2.3 deployment manifest](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-2.3/manifests/vanilla/vsphere-csi-driver.yaml#L350).
+
+- If the credentials used by CSI are changed, it will attempt to establish a connection to the Virtual Center every 60 seconds to prevent the account from getting locked.
```

Comment:
link to pull request will be good to refer here.

Reply:
done